vocabulary.md

```diff
@@ -9,19 +9,21 @@
-- **Conversation Event** - A conversation event signals changes in the conversational that may be acted upon.  Such an event may be at the beginning or ending of a Conversational Session, completion of a Conversation Processor, decoding of Conversation Information, changes to the state of a Conversation Endpoint, or changes to the status of a Conversation Stream.  Any component with access to the system is allowed to generate a Conversation Event.
-- **Conversation Facilitator** - coordinates communication between two or more Dialogue Systems and/or Processors during the course of one or more Sessions.  This allows dialogue Systems and associated Processors to collaborate regardless of technology being used.  Examples of Conversation Information include semantic, lexical, syntactic, and prosodic features. 
-- **Conversation Information Layer** - represents an abstraction of a type of information in a Dialog System.  A layer may be a specific type of acoustic, linguistic, non-linguistic, or paralinguistic features.  Examples of layers would be Cepstral features, Phonemes, Intonation Boundaries, Words, Phrases, Turn Boundaries, Syllabic Stress, Discourse Move Tiype, and specific Semantic representation schemes.
-- **Conversation Processors** - Conversation information is encoded and/or decoded by one or more Conversational Processors, also known as a Component.  Conversational Processors may also take as input the output from another Conversation Processor.   A Conversation Processor may generate Conversation Events and Conversation Streams. 
-- **Conversation Session** - A particular conversation that consists of two or more Conversation Streams (see below) generated by two or more agents through one or more Conversational Endpoints.   Sessions may be persistent, but they will often have a start-point and an end-point in time determined by one of the Agents or some other external event.
-- **Conversation Stream** - Each Conversational Endpoint generates one or more Conversation Streams based upon the capabilities of the Endpoint and the preferences of the Agent.  A Conversation Stream is associated with a particular Agent and may include any media type including text, audio, video, and application UI events.  
-- **Conversational Agent** - A digital participant in a conversation.   This may be an application with a consistent persona, such as Amazon Alexa, Google Assistant, the Target Google Assistant Action, a Facebook Messenger chatbot, or an IVR system at a bank.  Conversational agent is a common term utilized in the area of Dialogue System research and university level instruction; the term often is used to describe a human participant in a conversation.  For clarity of reference, however, the OVN will use the term "user" to identify a human participant.  (See "user" below.)
+- **Conversation** - a collaborative linguistic act between two or more conversational agents to achieve a user goal.
```

Comment:
Conversational Agent is defined below as a "digital participant". Is the intent to restrict /conversation/ to digital agents as well? (I would find this confusing.)